code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -296,3 +296,10 @@
 /obj/item/weapon/storage/box/syndie_kit/mimery/PopulateContents()
 	new /obj/item/weapon/spellbook/oneuse/mimery_blockade(src)
 	new /obj/item/weapon/spellbook/oneuse/mimery_guns(src)
+
+/obj/item/weapon/storage/box/syndie_kit/holoparasite
+	name = "box"
+
+/obj/item/weapon/storage/box/syndie_kit/holoparasite/PopulateContents()
+	new /obj/item/weapon/guardiancreator/tech/choose/traitor(src)
```

Comment:
Actually I think the "traitor" bit just makes it so it can't choose dextrous, which is fair. I may be wrong, though. It worked normally in testing